studio/pages/new/[slug].tsx

```diff
@@ -169,6 +169,15 @@ const Wizard: NextPageWithLayout = () => {
       kps_enabled: kpsEnabled,
     }
     if (postgresVersion) {
+      if (!postgresVersion.match(/1[2-9]\..*/)) {
+        setNewProjectLoading(false)
+        ui.setNotification({
+          category: 'error',
+          message: `Invalid Postgres Version`,
```

Comment:
error should indicate what the version format was expected to be

Reply:
Improved error message, feel free to make a suggestion if it's not clear enough 👍